### DISCUSSION TITLE
Car brand classification

### DISCUSSION THREAD

**Discussion by AHazimy** [ultralytics/yolov5#9480] [Q&A]
I am using yolov5 for object detection with my own dataset, my question is "YOLOV5 can classify cars by their brands" ?
Thanks in advance

**Comment by glenn-jocher** [comment]
Yes.

**Comment by spacewalk01** [comment]
You can classify cars by their brand, color and type. Here is an example dataset and code for classifying cars by their brand: https://www.kaggle.com/code/jeongbinpark/predict-the-car-brand-by-cnn
